torchmetrics/functional/text/wer.py

```diff
@@ -0,0 +1,29 @@
+from typing import Any
+
+from jiwer import compute_measures
+
+
+def wer(target: Any, preds: Any, concatenate_texts: bool = False) -> float:
```

Comment:
can we be more specific about the type? also need to be consistent between doc string a variable naming
```suggestion
def wer(references: List[str], predictions: List[str], concatenate_texts: bool = False) -> float:
```

Reply:
Fixed